test/functional/ui_test.rb

```diff
@@ -130,6 +139,15 @@ def show_spaces(str)
         assert_exit_code 0, run_result
       end
     end
+
+    describe "detect command" do
+      let(:result) { inspec("detect --no-color") }
+
+      it "has no color in the output" do
+        _(result.stdout).must_include("\e[1m\e[0m")
```

Comment:
I think the test should be that it does not include `"\e[1m` - in other words, ther should be no color control codes in the output.

Reply:
That's right. I confused no color with reset. Will make this change.